src/Avalonia.Controls/SelectableTextBlock.cs

```diff
@@ -32,6 +33,9 @@ public class SelectableTextBlock : TextBlock, IInlineHost
         public static readonly StyledProperty<IBrush?> SelectionBrushProperty =
             TextBox.SelectionBrushProperty.AddOwner<SelectableTextBlock>();
 
+        public static readonly StyledProperty<IBrush?> SelectionForegroundBrushProperty =
```

Comment:
I think the naming is incorrect here and below for compatiblity. I suppose technically it is more generic to say "Foreground" however since it really is just for text, the standard name should be used: `SelectionTextBrush`

 * See [TextBoxBase.SelectionTextBrush Property](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.textboxbase.selectiontextbrush?view=windowsdesktop-7.0)
 * Also [TextBoxBase.SelectionBrush](https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.textboxbase.selectionbrush?view=windowsdesktop-7.0) which already is the same.

Reply:
This uses the same naming as TextBox. We can change this in a future version.